libs/blocks/tabs/tabs.js

```diff
@@ -267,36 +276,41 @@ const init = (block) => {
   const tabContent = createTag('div', { class: 'tab-content' }, tabContentContainer);
   block.append(tabContent);
 
+  const isRadio = block.classList.contains('radio');
   // Tab List
   const tabList = rows[0];
   tabList.classList.add('tabList');
-  tabList.setAttribute('role', 'tablist');
+  tabList.setAttribute('role', isRadio ? 'radiogroup' : 'tablist');
   const tabListContainer = tabList.querySelector(':scope > div');
   tabListContainer.classList.add('tab-list-container');
+  const tabListLabel = config.pretext;
+  if (tabListLabel)tabList.setAttribute('aria-label', tabListLabel);
```

Comment:
Nit, this looks like something ESLint would complain about
```suggestion
  if (tabListLabel) tabList.setAttribute('aria-label', tabListLabel);
```